isis/src/control/apps/jigsaw/jigsaw.cpp

```diff
@@ -202,6 +202,16 @@ namespace Isis {
       throw IException(IException::User, msg, _FILEINFO_);
     }
 
+    if (ui.GetBoolean("TWIST") && ui.GetBoolean("RADIUS") && 
+        !ui.WasEntered("POINT_RADIUS_SIGMA")) {
+      string msg = "If solving for both twist and radius, must set a value "
+        "for point_radius_sigma, as otherwise the problem is under-constrained and a "
+        "failure may result.";
```

Comment:
Should this value be set to a default? It seems weird that enabling it but failing to set it would be allowed. Currently, ISIS doesn't set any value but defaults the sigma0 to 1e-10. Maybe this could share that default? Just a thought